Note on long build time (#214)
microsoft/presidio: docs/install.md

docker run --rm --name presidio-api --network mynetwork -d -p 8080:8080 -e WEB_PORT=8080 -e ANALYZER_SVC_ADDRESS=presidio-analyzer:3000 -e ANONYMIZER_SVC_ADDRESS=presidio-anonymizer:3001 -e RECOGNIZERS_STORE_SVC_ADDRESS=presidio-recognizers-store:3004 ${DOCKER_REGISTRY}/presidio-api:${PRESIDIO_LABEL}
```

> **NOTE: Building the deps images currently takes a lot of time** (~70 minutes, depending on the build machine). We are working on improving the build time through improving the build and providing pre-built dependencies.

Reviewer: ```suggestion
**NOTE: Building the deps images currently takes some time** (~70 minutes, depending on the build machine). We are working on improving the build time through improving the build and providing pre-built dependencies.
```
